### PR TITLE
Add documentation on the land-use emulator

### PR DESCRIPTION
**Pull request by OFR-IIASA**
In order to review, please make a local build of the documentation and read /message_doc/_build/html/land_use/emulator.html

### CODE REVIEW

File: land_use/emulator.rst [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+
+Equations and constraints
+-------------------------
+
+The land-use pathways are integrated into MESSAGE using a dedicated set of equations (add link to documentation). At the core, these state that the linear combination of land-use pathways must be equal to 1. Therefore, separately for each region, discrete land-use development pathways can either be used individually or combine linearly to obtain biomass quantities for example, which are not explicitly represented as part of the land-use matrix. This also applies to the mitigation dimension i.e. to the GHG categories.
```
Comment (behnam-zakeri) [inline land_use/emulator.rst:27]: This sentence "At the core, these state that the linear combination of land-use pathways must be equal to 1. Therefore, separately" and the following discussion is not clear. Is this the sum of land shares from a total land available?

**Review by behnam-zakeri — CHANGES_REQUESTED** [review]

**Comment by khaeru** [comment]
Per discussion in Slack, the 2020 documentation will be released now that #51 is in. This PR should be (a) rebased, (b) updated with GLOBIOM documentation from the colleagues who will supply it, and (c) merged in advance of a 2021 release.

**Comment by OFR-IIASA** [comment]
A few comments from a conversation on slack (from Behnam, Oct. 13, 2020) to improve the documentation of the emulator:
- the legend and labels of some figures can be improved by a more suitable size, also for the figure captions adding more information on each sub-plot when there are multiple figures. (e.g., figure (above) shows..., while below shows...).
- does it make sense to add numbers to equations (Eq. 1, ...)? Then, it's easier to refer to them in the text.
- the costs and prices are in $2005. It may be worth to update this as we are in 2020 and it may be difficult to back-calculate the numbers.

**Comment by OFR-IIASA** [comment]
@behnam-zakeri I have made changes to address the last set of comments:
- Adjusting all figures and describing the panels in the text
- Switching all units to $2010
- Adding formula references

**Review by khaeru — APPROVED** [review]

**Comment by khaeru** [comment]
Compiles with no warnings. Thanks @OFR-IIASA!

**Pull request merged**